tests/test_cli.py

```diff
@@ -40,22 +54,25 @@
     "lambda.nim",  # https://github.com/adsharma/py2many/issues/27
     "lambda.rs",  # https://github.com/adsharma/py2many/issues/15
     "str_enum.jl",  # https://github.com/adsharma/py2many/issues/26
-]
+] + [f"{case}.cpp" for case in TEST_CASES]
+# See https://github.com/adsharma/py2many/issues/24 for cpp failures
+
+
+def has_main(filename):
```

Comment:
Yeah - this is simple and effective. Good for now.

In the long run, handling this via a rename pass would avoid any issues caused by dealing with strings vs AST

Reply:
In the long run we do https://github.com/adsharma/py2many/issues/25 and https://github.com/adsharma/py2many/issues/30, and this `has_main` gets removed.